frontend/public/style/_common.scss

```diff
@@ -145,6 +145,10 @@
   border-radius: 2px;
 }
 
+.co-pre-line-wrap {
```

Comment:
.co-pre-line, otherwise it's not clear if it's pre-line or pre-wrap since both are in the name.

Reply:
Fixed these last comments -thanks!